torch/_inductor/lowering.py

```diff
@@ -1627,8 +1627,8 @@ def inner_fn(index):
     )
 
 
-@register_lowering(inductor_prims._bucketize, type_promotion_kind=None)
-def _inductor_bucketize(
+@register_lowering(aten.bucketize, type_promotion_kind=None)
```

Comment:
Out of curiosity: do we have data on the current Triton lowering's perf compared to `aten.bucketize`?

Reply:
In the comments of #104456 we're showing better performance than aten.bucketize (.30ms eager, .22ms PT2), on a specific example (2^24 float32 values, 1024 buckets)